DSCResources/MSFT_xServiceResource/MSFT_xServiceResource.psm1

```diff
@@ -322,12 +327,12 @@ function Set-TargetResource
     {
          if(( $PSBoundParameters.ContainsKey("State")) -and ($State -eq "Running"))
         {
-            StartService $svc
+            StartService $svc $StartupTimeout
```

Comment:
Please don't use positional parameters. Specify parameter names and their values explicitly